project-metrics/metrics_service/apis/codecov_test.py

```diff
@@ -0,0 +1,46 @@
+"""Tests for Codecov API interface."""
+
+from agithub import base as agithub_base
+import unittest
+from unittest import mock
+
+import env
+from apis import codecov
+
+
+class TestCodecovApi(unittest.TestCase):
+
+  def setUp(self):
+    super(TestCodecovApi, self).setUp()
+
+    mock.patch.object(
+        env, 'get', side_effect={
+            'GITHUB_REPO': '__repo__',
+        }.get).start()
+    self.mock_request = mock.patch.object(
+        agithub_base.Client, 'request', autospec=True).start()
+    self.addCleanup(mock.patch.stopall)
+
+    self.codecov_api = codecov.CodecovApi()
+
+  def testGetAbsoluteCoverage(self):
+    self.mock_request.return_value = (200, {
+        'commit': {
+            'totals': {
+                'c': '13.37'
```

Comment:
nit: comma if you use this format. Without a comma this should go in one line: `'totals': {'c': '13.37'},` ...

Reply:
Updated; I use the autoformatter generally, but sometimes I does stuff like this and I have to disable it :unamused: